src/NuGet.Core/NuGet.Commands/RestoreCommand/Utility/AuditUtility.cs

```diff
@@ -272,6 +274,11 @@ private static (string severityLabel, NuGetLogCode code) GetSeverityLabelAndCode
                                 continue;
                             }
 
+                            if (SuppressedAdvisories.Contains(knownVulnerability.Url.ToString()))
```

Comment:
```suggestion
                            if (SuppressedAdvisories.Contains(knownVulnerability.Url.OriginalString))
```

Unfortunately it was a big mistake for me to set `Url` to `System.Uri` rather than `string`. But we can still avoid obvious allocations.

Reply:
Fixed